src/code/entities/bank4.asm

```diff
@@ -188,71 +188,88 @@ func_004_6E1D::
     ld   hl, wEntitiesPrivateState3Table          ; $6E2F: $21 $D0 $C2
     jp   AddEntitySpeedToPos_04.updatePosition    ; $6E32: $C3 $EF $6D
 
-func_004_6E35::
+; Inputs:
+;   bc   entity index
+;
+; Outputs:
+;   d   x distance (Link's position - entity's position)
+;   e   0x01 if Link is to the left of the entity, 0x00 otherwise
+GetEntityXDistanceToLink_04::
     ld   e, $00                                   ; $6E35: $1E $00
     ldh  a, [hLinkPositionX]                      ; $6E37: $F0 $98
     ld   hl, wEntitiesPosXTable                   ; $6E39: $21 $00 $C2
     add  hl, bc                                   ; $6E3C: $09
     sub  [hl]                                     ; $6E3D: $96
     bit  7, a                                     ; $6E3E: $CB $7F
-    jr   z, .jr_6E43                              ; $6E40: $28 $01
+    jr   z, .positive                             ; $6E40: $28 $01
 
     inc  e                                        ; $6E42: $1C
 
-.jr_6E43
+.positive
     ld   d, a                                     ; $6E43: $57
     ret                                           ; $6E44: $C9
 
-func_004_6E45::
+; Inputs:
+;   bc   entity index
+;
+; Outputs:
+;   d   y distance (Link's position - entity's position)
+;   e   0x02 if Link is above the entity, 0x03 otherwise
+GetEntityYDistanceToLink_04::
     ld   e, $02                                   ; $6E45: $1E $02
```

Comment:
How about using the constants here?

```suggestion
;   e   DIRECTION_UP if Link is above the entity, DIRECTION_DOWN otherwise
GetEntityYDistanceToLink_04::
    ld   e, DIRECTION_UP                          ; $6E45: $1E $02
```

Reply:
Sure, that makes sense. I'll go ahead and change that.